src/Neo/SmartContract/Native/PolicyContract.cs

```diff
@@ -181,6 +191,17 @@ public uint GetMaxValidUntilBlockIncrement(IReadOnlyStore snapshot)
             return (uint)(BigInteger)snapshot[_maxValidUntilBlockIncrement];
         }
 
+        /// <summary>
+        /// Gets the length of the chain accessible to smart contracts.
+        /// </summary>
+        /// <param name="snapshot">The snapshot used to read data.</param>
+        /// <returns>MaxTraceableBlocks value.</returns>
+        [ContractMethod(Hardfork.HF_Echidna, CpuFee = 1 << 15, RequiredCallFlags = CallFlags.ReadStates)]
+        public uint GetMaxTraceableBlocks(IReadOnlyStore snapshot)
```

Comment:
price is high, but i guess no one will ever really need it, so chill.

Reply:
14 is good for me

Reply:
All other getters use `1<<15`, so I'd use `1<<15` for now for this getter and leave the question to #3894.